app/models/pregnancy.rb

```diff
@@ -45,6 +45,9 @@ class Pregnancy
   # Procedure result - generally for administrative use
   field :fax_received, type: Boolean
   field :procedure_cost, type: Integer
+  field :patient_contribution, type: Integer
```

Comment:
Can you add a comment after these three fields to make a note that they're temporary stand-ins for `pledge` objects? Something like `field :patient_contribution, type: Integer # temporary` will do nicely!